Library/Formula/libcdr.rb

```diff
@@ -0,0 +1,30 @@
+class Libcdr < Formula
+  homepage "https://wiki.documentfoundation.org/DLP/Libraries/libcdr"
+  url "http://dev-www.libreoffice.org/src/libcdr/libcdr-0.1.1.tar.bz2"
+  sha256 "72fe7bbbf2275242acdf67ad2f9b6c71ac9146a56c409def360dabcac5695b49"
+
+  depends_on "pkg-config" => :build
+  depends_on "boost" => :build
+  depends_on "icu4c"
+  depends_on "librevenge"
+  depends_on "little-cms2"
+
+  def install
+    ENV.cxx11
+    system "./configure", "--disable-werror",
+                          "--without-docs",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    ('test.cpp').write <<-EOS.undent
```

Comment:
Sorry, misunderstanding here. I meant ` (testpath/"test.cpp") ` rather than just ` ('test.cpp') `. I was trying to say double quotes rather than single; I should have just said that in hindsight :smile_cat: